### PR TITLE
Fix ci errors

### PR DESCRIPTION
**Pull request by milmazz**
## Description

* Fix syntax error in `Makefile` when publishing code coverage results 
* Use development version of InchEx, this is needed to avoid the following error:

```
(InchEx.Docs.Retriever.Error) module Mix.Tasks.Scenic.Hash was not compiled with flag --docs
```

## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
  not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature
  but make things better)

## Checklist

- [ ] Check other PRs and make sure that the changes are not done yet.
- [ ] The PR title is no longer than 64 characters.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,4 +14,4 @@ script:
 after_script:
   - make docs_report
 after_success:
-  - make push_docs_report
+  - bash <(curl -s https://codecov.io/bash)
```
Comment (yordis) [inline .travis.yml]: @milmazz can't be on the `Makefile`?

File: mix.exs [hunk 2]
```diff
@@ -99,7 +99,7 @@ defmodule Scenic.Mixfile do
       {:ex_doc, ">= 0.0.0", only: [:dev]},
       {:credo, ">= 0.0.0", only: [:dev, :test], runtime: false},
       {:excoveralls, ">= 0.0.0", only: :test, runtime: false},
-      {:inch_ex, ">= 0.0.0", only: :dev, runtime: false},
+      {:inch_ex, github: "rrrene/inch_ex", only: [:dev, :test], runtime: false},
```
Comment (yordis) [inline mix.exs]: Why we point to the Github version instead of Hex?!

File: .travis.yml (hunk 1 above)
Comment (milmazz) [reply .travis.yml]: @yordis At least as is no, is a syntax error.

File: mix.exs (hunk 2 above)
Comment (milmazz) [reply mix.exs]: @yordis No, with the latest release (`1.0.0`) published on Hex I'm getting the following error:

```
(InchEx.Docs.Retriever.Error) module Mix.Tasks.Scenic.Hash was not compiled with flag --docs
```

You can look at previous Travis CI build logs.

But, @rrrene seems to have fixed that error already on `master`, he is working on the `2.0.0` version there.

Comment (boydm) [reply mix.exs]: I'm going to take this (even tho I know the inch_ex report is going to be _terrible_).

We just have to remember to switch it to 2.0.0 when rrrene releases it.

**Pull request merged**